pkg/osquery/tables/execparsers/apt/parser_test.go

```diff
@@ -0,0 +1,132 @@
+					"package":         "accountsservice",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "22.07.5-2ubuntu1.4",
+					"current_version": "22.07.5-2ubuntu1.3",
+				},
+				{
+					"package":         "apt-utils",
+					"sources":         "jammy-updates",
+					"update_version":  "2.4.9",
+					"current_version": "2.4.8",
+				},
+				{
+					"package":         "apt",
+					"sources":         "jammy-updates",
+					"update_version":  "2.4.9",
+					"current_version": "2.4.8",
+				},
+				{
+					"package":         "base-files",
+					"sources":         "jammy-updates",
+					"update_version":  "12ubuntu4.3",
+					"current_version": "12ubuntu4.2",
+				},
+				{
+					"package":         "binutils-common",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "2.38-4ubuntu2.2",
+					"current_version": "2.38-4ubuntu2.1",
+				},
+				{
+					"package":         "binutils-x86-64-linux-gnu",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "2.38-4ubuntu2.2",
+					"current_version": "2.38-4ubuntu2.1",
+				},
+				{
+					"package":         "dpkg",
+					"sources":         "jammy-updates",
+					"update_version":  "1.21.1ubuntu2.2",
+					"current_version": "1.21.1ubuntu2.1",
+				},
+				{
+					"package":         "libkrb5-3",
+					"sources":         "jammy-updates",
+					"update_version":  "1.19.2-2ubuntu0.2",
+					"current_version": "1.19.2-2ubuntu0.1",
+				},
+				{
+					"package":         "libldap-common",
+					"sources":         "jammy-updates,jammy-updates",
+					"update_version":  "2.5.14+dfsg-0ubuntu0.22.04.2",
+					"current_version": "2.5.13+dfsg-0ubuntu0.22.04.1",
+				},
+				{
+					"package":         "openssl",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "3.0.2-0ubuntu1.10",
+					"current_version": "3.0.2-0ubuntu1.8",
+				},
+				{
+					"package":         "perl-base",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "5.34.0-3ubuntu1.2",
+					"current_version": "5.34.0-3ubuntu1.1",
+				},
+				{
+					"package":         "perl-modules-5.34",
+					"sources":         "jammy-updates,jammy-updates,jammy-security,jammy-security",
+					"update_version":  "5.34.0-3ubuntu1.2",
+					"current_version": "5.34.0-3ubuntu1.1",
+				},
+				{
+					"package":         "sudo",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "1.9.9-1ubuntu2.4",
+					"current_version": "1.9.9-1ubuntu2.3",
+				},
+				{
+					"package":         "vim",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "2:8.2.3995-1ubuntu2.9",
+					"current_version": "2:8.2.3995-1ubuntu2.3",
+				},
+				{
+					"package":         "xxd",
+					"sources":         "jammy-updates,jammy-security",
+					"update_version":  "2:8.2.3995-1ubuntu2.9",
+					"current_version": "2:8.2.3995-1ubuntu2.3",
+				},
+			},
+		},
+	}
+
+	for _, tt := range tests {
```

Comment:
Note that you only have one test. So the table test pattern is a little bit of overkill. It's fine though -- it makes it easy to add more, if we need that

Reply:
I also prefer table pattern even with just one case. As other said, it makes it easier to add stuff in the future, but it is also just our consistent pattern. It now throws me off a little when I come across a non-table test =).

Reply:
Added more tests :)